201-logic-app-transform-function/azuredeploy.json

```diff
@@ -0,0 +1,49 @@
+{
+    "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "existingFunctionAppName": {
+            "type": "string",
+            "metadata": {
+                "description": "The name of the function container in which the function has to be created."
+            }
+        },
+        "functionName": {
+            "type": "string",
+            "defaultValue": "transformfunction",
+            "metadata": {
+                "description": "The name of the function."
+            }
+        }
+    },
+    "resources": [{
+        "apiVersion": "2015-08-01",
+        "name": "[concat(parameters('existingFunctionAppName'),'/', parameters('functionName'))]",
+        "type": "Microsoft.Web/sites/functions",
+        "properties": {
+            "config": {
+                "bindings": [{
+                    "name": "req",
+                    "webHookType": "genericJson",
+                    "direction": "in",
+                    "type": "httpTrigger"
+                }, {
+                    "name": "res",
+                    "direction": "out",
+                    "type": "http"
+                }]
+            },
+            "files": {
+                "IntegrationXsltException.csx": "//-----------------------------------------------------------------------\r\n// <copyright file=\"IntegrationXsltException.csx\" company=\"Microsoft\">\r\n//      All rights reserved.\r\n// </copyright>\r\n//-----------------------------------------------------------------------\r\n\r\nusing System;\r\nusing System.Net;\r\n\r\n[System.Diagnostics.CodeAnalysis.SuppressMessage(\"Microsoft.Design\", \"CA1032:ImplementStandardExceptionConstructors\", Justification = \"Not needed.\")]\r\n[System.Diagnostics.CodeAnalysis.SuppressMessage(\"Microsoft.Usage\", \"CA2237:MarkISerializableTypesWithSerializable\", Justification = \"Not needed.\")]\r\npublic class IntegrationXsltException : Exception\r\n{\r\n    public IntegrationXsltException()\r\n    {\r\n    }\r\n\r\n    public IntegrationXsltException(string message) : base(message, null)\r\n    {\r\n        this.StatusCode = HttpStatusCode.InternalServerError;\r\n    }\r\n\r\n    public IntegrationXsltException(HttpStatusCode statusCode, string message)\r\n        : base(message)\r\n    {\r\n        this.StatusCode = statusCode;\r\n    }\r\n\r\n    public IntegrationXsltException(HttpStatusCode statusCode, string message, Exception innerException)\r\n        : base(message, innerException)\r\n    {\r\n        this.StatusCode = statusCode;\r\n    }\r\n\r\n    public HttpStatusCode StatusCode { get; set; }\r\n}",
```

Comment:
Azure functions provide a way to execute small pieces of code on the cloud in a sandbox environment.

The purpose of this template is to create one such function with XSLT trasnfrom capabilities for logic apps integration scenarios.

In this template we are specifying the files and their code and that gets executed..